main.tf

```diff
@@ -352,6 +352,11 @@ data "aws_iam_policy_document" "agentless_scan_task_policy_document" {
       variable = "iam:ResourceTag/LWTAG_SIDEKICK"
       values   = ["*"]
     }
+    condition {
```

Comment:
is it possible to also set the resource to something describing the task role, or is that totally unknown?

Reply:
The resource name has some variance, for example people can set their own path-prefix/name prefix/name suffix, and then this is per-account. Due to this variance you do not gain much compared to what is already gained by requiring the tag and the service.